src/appimagetool/appimagetool.go

```diff
@@ -778,95 +778,3 @@ func GenerateAppImage(appdir string) {
 	fmt.Println("at https://github.com/AppImage/appimage.github.io")
 }
 
-
-// main Command Line Entrypoint. Defines the command line structure
```

Comment:
Is it correct that this PR wants to remove this whole section?

Reply:
uh oh; this commit has been included in #104; shall I close this?; looks like my mistake

Reply:
The net change, I wanted was L576 smh; maybe I should rewrite this